person.rb

```diff
@@ -1,17 +1,24 @@
-class Person
+require_relative 'Nameable'
```

Comment:
- Please kindly refer to the correct file name which is **nameable**. This issue produces an error when running this file. Let's kindly fix :wrench: it before merging our PR

Reply:
Done sir...
@FrederickMih